docs/writing-tests.md

```diff
@@ -1,32 +1,12 @@
 # Writing tests
 
-## Adding new tests
+## Alerting in Icinga
 
-Tests that are supposed to be run by Icinga also have to be added to the file
-`modules/monitoring/manifests/checks/smokey.pp` in the [govuk-puppet](https://github.com/alphagov/govuk-puppet) repository. For
-example, the test [frontend.feature](/features/frontend.feature)
-is added to Icinga like this:
+In order to alert on a failing scenario in Icinga, you need to:
 
-```puppet
-icinga::check_feature {
-  'check_frontend':          feature => 'frontend';
-  #other feature tests
-}
-```
+- Ensure the feature is [listed in the Puppet config](https://github.com/alphagov/govuk-puppet/blob/d7af16e96aed682facb5cf5bc3e3972510c64ca2/hieradata_aws/integration.yaml#L378)
 
-## Prioritising scenarios
+- Tag the scenario using "@<tag>" (tags are listed [here](https://github.com/alphagov/govuk-puppet/blob/4ec2949332079570be5493753bb84a983218a444/modules/icinga/manifests/check_feature.pp))
```

Comment:
Have improved the [inaccessible `here` link](https://webaim.org/techniques/hypertext/link_text#uninformative) and described it a little:

```suggestion
- Tag the scenario with a [priority "@<tag>"](https://github.com/alphagov/govuk-puppet/blob/4ec2949332079570be5493753bb84a983218a444/modules/icinga/manifests/check_feature.pp):
```

Reply:
Sure, updated.